core/unix/loader.c

```diff
@@ -1574,7 +1574,7 @@ dynamorio_lib_gap_empty(void)
         while (memquery_iterator_next(&iter) && iter.vm_start < dr_end) {
             if (iter.vm_start >= dr_start && iter.vm_end <= dr_end &&
                 iter.comment[0] != '\0' &&
-                strstr(iter.comment, DYNAMORIO_LIBRARY_NAME) == NULL) {
+                strstr(iter.comment, get_dynamorio_library_path()) == NULL) {
```

Comment:
So this is now the full path: does `strstr` still make sense or should it be `strcmp`?

Reply:
Good point. I think it could be strcmp, i'll check and resolve if true.

Reply:
Done.